integreat_cms/cms/views/analytics/translation_coverage_view.py

```diff
@@ -44,67 +45,48 @@ def get(self, request, *args, **kwargs):
 
-            # Get the QuerySet of translations which are currently not in translation (so either up-to-date or outdated)
-            currently_not_in_translation = page_translations.filter(
-                currently_in_translation=False
-            )
-            # Get a list of translations which are outdated
-            outdated_page_translations = [
-                translation
-                for translation in currently_not_in_translation
-                if translation.is_outdated
-            ]
-            # Count the words in the outdated translations
-            outdated_word_count[language] = sum(
-                len(translation.content.split())
-                for translation in outdated_page_translations
-            )
-            # Count the outdated translations
-            outdated_count = len(outdated_page_translations)
-            # Append the translation coverage data for this language
-            translation_coverage_data.append(
-                {
-                    # The number of pages which do not have a translation in this language
-                    translation_status.MISSING: (
-                        total_number_pages - page_translations.count()
-                    ),
-                    # The number of translations which are outdated
-                    translation_status.OUTDATED: outdated_count,
-                    # The number of translations which are currently being translated
-                    translation_status.IN_TRANSLATION: (
-                        page_translations.count() - currently_not_in_translation.count()
-                    ),
-                    # The number of up-to-date translations (neither missing, nor currently in translation, nor outdated)
-                    translation_status.UP_TO_DATE: (
-                        currently_not_in_translation.count() - outdated_count
-                    ),
-                }
-            )
+                if translation_state == translation_status.OUTDATED:
+                    translation = page.get_translation(language.slug)
+                    outdated_word_count[language] += len(translation.content.split())
+            translation_coverage_data[language] = language_coverage_data
+
+        for (language, data) in translation_coverage_data.items():
             logger.debug(
-                "Coverage data for %r: %r", language, translation_coverage_data[-1]
+                "Coverage data for %r: %r",
+                language,
+                data,
             )
-
         logger.debug("Outdated word count: %r", outdated_word_count)
 
         # Assemble the ChartData in the format expected by ChartJS (one dataset for each translation status)
         chart_data = {
             "labels": [
-                language.translated_name for language in region.active_languages
+                language.translated_name
+                for language in translation_coverage_data.keys()
```

Comment:
```suggestion
                language.translated_name for language in translation_coverage_data
```
Ok, I'm very sorry, I thought `keys()` would be more readable, but if pylint prefers the direct dict iteration, maybe that's better code style :information_desk_person: